dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -698,7 +698,19 @@ private void checkToolTokens(User authUser, Long userId, String organization) {
     @ApiOperation(value = "Get the authenticated user's starred workflows.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Entry.class, responseContainer = "List")
     public Set<Entry> getStarredWorkflows(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User user) {
         User u = userDAO.findById(user.getId());
-        return u.getStarredEntries().stream().filter(element -> element instanceof Workflow)
+        return u.getStarredEntries().stream().filter(element -> element instanceof BioWorkflow)
```

Comment:
I don't think it's worth it, but I'm going to throw it out there -- ideally we would have the database do the filtering instead of the web service. Currently the most starred entries by a single user is 25, so it won't make a a huge difference, but something to keep in mind for future PRs.